packages/content-handler/test/newsletter.test.ts

```diff
@@ -129,6 +130,41 @@ describe('Newsletter email test', () => {
       expect(handler).to.be.undefined
     })
 
+    it('returns SubstackHandler for substack newsletter with static tweets', async () => {
+      const html = load(
+        './test/data/substack-with-static-tweets-newsletter.html'
+      )
+      const handler = await getNewsletterHandler({
+        html,
+        postHeader: '',
+        from: '',
+        unSubHeader: '',
+      })
+      expect(handler).to.be.instanceOf(SubstackHandler)
+    })
+
+    it('fixes up static tweets in Substack newsletters', async () => {
+      const url = 'https://astralcodexten.substack.com/p/nick-cammarata-on-jhana'
+      const html = load(
+        './test/data/substack-with-static-tweets-newsletter.html'
+      )
+      const handler = await getNewsletterHandler({
+        html,
+        postHeader: '',
+        from: '',
+        unSubHeader: '',
+      })
+      expect(handler).to.be.instanceOf(SubstackHandler)
+
+      const dom = parseHTML(html).document
+      expect(handler?.shouldPreParse(url, dom)).to.be
```

Comment:
I guess we left out the expected result here

Reply:
I think this verifies that its true, but I'll make it more clear.